Assets/_BForBoss/_Character/InputActions/BFB_FirstPerson_InputActions.inputactions

```diff
@@ -76,6 +76,14 @@
                     "expectedControlType": "Button",
```

Comment:
I think we should move the input actions asset into the Core folder, since its used by both _Weapon and _Character. Or look at breaking it up somehow

Reply:
This doesn't need to happen as part of this pr though necessarily

Reply:
yes you are correct.